pysat/_instrument.py

```diff
@@ -412,8 +414,9 @@ def __getitem__(self, key):
                 try:
                     # integer based indexing
                     return self.data.iloc[key]
-                except TypeError:
+                except (TypeError, ValueError):
                     # If it's not an integer, TypeError is thrown
+                    # If it's a list, ValueError is thrown
```

Comment:
@rstoneback, this should fix #504.  My initial suggestion is on the right track, but pointed tot he wrong spot.

Reply:
Thanks! I appreciate the update. I won't be able to directly check, in a processing context, until tomorrow at least.

Reply:
I've added a unit test to this branch, which is working locally